### PR TITLE
Removed empty cell

### LINKED ISSUE #3206: Blank Cell in Documentaion
In reviewing docs for 0.41.0 release, found a blank cell that can be removed : https://evalml.alteryx.com/en/v0.41.0/demos/text_input.html.

image

### PR DESCRIPTION
**Pull request by Mahesh1822**
### Pull Request Description
Removed empty cell from docs/source/demos/text_input.ipynb  "Fixes #3206"

### CODE REVIEW

File: docs/source/release_notes.rst
Comment (jeremyliweishih) [inline docs/source/release_notes.rst:10]: Could you add some more color here? Something like, "Removed empty cell in text_input.ipynb  :pr:`3234`" would be great!

**Comment by jeremyliweishih** [comment]
Thanks for putting up this new PR @Mahesh1822, just had one comment and I can approve after. In the future you do not need to put up a new PR! You can just make changes to the branch you're on, push the branch up, and the change will be in your PR as well. Let me know if that makes sense and how else I can help. Since a commit has been merged into our `main` (base) branch, you will need to update this branch by merging in the latest changes from main as well before you can merge this PR.

**Comment by jeremyliweishih** [comment]
Another handy thing to note is: if you add "Fixes #3206" in your PR description, it will automatically close that issue once this PR goes in. Heres some documentation from github on how that works: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue!

**Comment by Mahesh1822** [comment]
Thanks I did the changes and updated my repo too. Please check if everything is right.

**Review by jeremyliweishih — APPROVED** [review]
Thank you this is great! Should be good once tests pass!

**Comment by Mahesh1822** [comment]
It's showing a conflict. Why is it?

**Comment by Mahesh1822** [comment]
`<<<<<<< remove_cell
	* Removed empty cell in text_input.ipynb :pr:`3234`
=======
        * Removed potential prediction explanations failure when pipelines predicted a class with probability 1 :pr:`3221`
>>>>>>> main`

**Comment by Mahesh1822** [comment]
<<<<<<< remove_cell
	* Removed empty cell in text_input.ipynb :pr:`3234`
=======
        * Removed potential prediction explanations failure when pipelines predicted a class with probability 1 :pr:`3221`
>>>>>>> main

**Comment by Mahesh1822** [comment]
I edited the doc a bit. And it solved the conflict.

**Comment by jeremyliweishih** [comment]
@Mahesh1822 in general a conflict arises if someone else merged a change into `main` that touches the files in this PR. I didn't see the exact conflict but I imagine it was because someone made a change in main and had a change in the release notes that coincided with yours. Thanks for fixing the conflict!

**Comment by chukarsten** [comment]
Thanks for contributing @Mahesh1822 !

**Comment by jeremyliweishih** [comment]
@Mahesh1822 I'm going to merge before another conflict arises. Thank you!

**Comment by Mahesh1822** [comment]
Here goes my first contribution : )

**Pull request merged**
